dotnet/src/InternalUtilities/src/Schema/JsonSchemaMapper.ReflectionHelpers.cs

```diff
@@ -170,16 +170,16 @@ private static bool TryGetStringEnumConverterValues(JsonTypeInfo typeInfo, JsonC
     {
         public ParameterLookupKey(string name, Type type)
         {
-            Name = name;
-            Type = type;
+            this.Name = name;
+            this.Type = type;
```

Comment:
The changes to InternalUtilities are nice improvements, but it would be better to submit them to main directly.  It doesn't look like they are directly related to the pinecone changes, or did I miss something? I'm trying to keep the feature branch reasonably constrained so that I don't end up with difficult merges later on.

Reply:
these are generated automatically when running build.cmd - nothing to do with the Pinecone changes. Same thing happens on the main branch so I will do as you suggest and revert those changes from here.